code/Test_definitions/population-density-data.feature

```diff
@@ -0,0 +1,277 @@
+Feature: CAMARA Population Density Data API, v0.1.1
+    # Input to be provided by the implementation to the tester
+    #
+    # Implementation indications:
+    # * Geohash precisions allowed
+    # * Min start and end dates allowed
+    # * Max requested time period allowed
+    # * Max size of the response(Combination of area, startDate, endDate an precision requested) supported for a sync response
+    # * Max size of the response(Combination of area, startDate, endDate an precision requested) supported for an async response
+    # * Limitations about max complexity of requested area allowed
+    #
+    # Testing assets:
+    # * An Area within the supported region
+    # * An Area partially within the supported region
+    # * An Area outside the supported region
+    #
+    # References to OAS spec schemas refer to schemas specifies in population-density-data.yaml, version 0.1.1
+
+    Background: Common retrievePopulationDensity  setup
+        Given an environment at "apiRoot"
+        And the resource "/population-density-data/v1/retrieve"
+        And the header "Content-Type" is set to "application/json"
+        And the header "Authorization" is set to a valid access token
+        And the header "x-correlator" is set to a UUID value
+        And the request body is set by default to a request body compliant with the schema
+
+    # Happy path scenarios
+
+    @population_density_data_01_supported_area_success_scenario
+    Scenario: Validate success response for a supported area request
+        Given the request body property "$.area" is set to a valid testing area within supported regions
+        And the request body property "$.startDate" is set to a valid testing future date
+        And the request body property "$.endDate" is set to a valid testing future date later than body property "$.startDate"
+        When the request "retrievePopulationDensity" is sent
+        Then the response status code is 200
+        And the response header "Content-Type" is "application/json"
+        And the response header "x-correlator" has same value as the request header "x-correlator"
+        And the response body complies with the OAS schema at "/components/schemas/PopulationDensityResponse"
+        And the response property "$.status" value is "SUPPORTED_AREA"
+        And the response property "$.timedPopulationDensityData[*].startTime" is equal to or later than request body property "$.startDate"
+        And the response property "$.timedPopulationDensityData[*].endTime" is equal to or earlier than request body property "$.endDate"
+        And the response property "$.timedPopulationDensityData[*].cellPopulationDensityData[*].populationDensityData.dataType.datatype" is equal to "LOW_DENSITY" or "DENSITY_ESTIMATION"
```

Comment:
duplicate `datatype `no?
 should be `"$.timedPopulationDensityData[*].cellPopulationDensityData[*].populationDensityData.dataType" is equal to "LOW_DENSITY" or "DENSITY_ESTIMATION"`

Reply:
to be solved